test/system/dashboard_test.rb

```diff
@@ -17,9 +17,9 @@ class DashboardSystemTest < ApplicationSystemTestCase
     sign_in_as(user)
 
     visit dashboard_path
-    assert_no_text "You have not added any friends yet."
+    assert_no_text "You have not followed any user yet."
 
-    friends_heading = find :element, "h2", :text => "My friends"
+    friends_heading = find :element, "h2", :text => "Followed users"
```

Comment:
Maybe use  "Following" instead of "Followed users" for better clarity. It aligns with the common terminology users are familiar with, like "Following" and "Followers," which may feel more intuitive.